app/views/shared/shop_show/_shop_cover_photo.html.haml

```diff
@@ -0,0 +1,7 @@
+- content_for :header_fluid do
+
+  #shop-cover-photo.container-fluid
+    .cover_text
+      %h1.display-8=@shop.name
+      %p.lead Unit 32 /33, The Chq Building
```

Comment:
How do we get the location (text) that the user selected on the map? From the database, I can only get lat and longitudes..